CHANGELOG.rst

```diff
@@ -6,8 +6,9 @@ This document describes changes between each past release.
 1.1 (unreleased)
 ----------------
 
-- Nothing changed yet.
+### Fixed
 
+- Add *.wsgi in the manifest file (#237)
```

Comment:
It's here in the CHANGELOG entry that I was thinking about the rephrasing (I've seen you changed the title of the issue instead).